editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -415,22 +415,25 @@ const RowForBaseControl = React.memo((props: RowForBaseControlProps) => {
               alignItems: 'center',
               gap: 6,
               color:
-                dataPickerButtonData.popupIsOpen || isHovered || !isValueSet
+                dataPickerButtonData.popupIsOpen || isHovered || isConnectedToData
                   ? colorTheme.dynamicBlue.value
                   : undefined,
+              cursor: isHovered ? 'pointer' : 'default',
```

Comment:
I think this can just remain 'pointer' as this field only has effect when the mouse hovers

Reply:
pfft yeah, my brain stopped functioning here

Reply:
(https://github.com/concrete-utopia/utopia/pull/5569/commits/e0f318a2a765aceb6a5249af54b0dcc346865b3a)